src/components/GridTable.tsx

```diff
@@ -137,15 +137,17 @@ export const condensedStyle: GridStyle = {
 /** Renders each row as a card. */
 export const cardStyle: GridStyle = {
   ...defaultStyle,
-  firstNonHeaderRowCss: Css.$,
+  firstNonHeaderRowCss: Css.mt2.$,
   cellCss: Css.p2.my1.bt.bb.bGray400.$,
   firstCellCss: Css.bl.add({ borderTopLeftRadius: "4px", borderBottomLeftRadius: "4px" }).$,
   lastCellCss: Css.br.add({ borderTopRightRadius: "4px", borderBottomRightRadius: "4px" }).$,
-  rootCss: Css.dig.$,
```

Comment:
I'd wanted to try `inline-grid` but I think it's easier to get more deterministic widths if this is always `grid` and then callers just put it into a div of width 550 or what not.